server/dns-service/dnsservice-api/src/main/java/org/apache/james/dnsservice/api/InMemoryDNSService.java

```diff
@@ -17,19 +17,17 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.mpt.smtp.dns;
+package org.apache.james.dnsservice.api;
```

Comment:
why do you want in implementation in an api project ? I mean, why not in a test-jar, but an api is supposed to not ship any implementation, right ?